pkg/controller/volume/persistentvolume/scheduler_binder_test.go

```diff
@@ -1478,6 +1478,6 @@ func TestFindAssumeVolumes(t *testing.T) {
 		if !unboundSatisfied {
 			t.Errorf("Test failed: couldn't find PVs for all PVCs")
 		}
-		testEnv.validatePodCache(t, "after-assume", testNode.Name, pod, expectedBindings, []*v1.PersistentVolumeClaim{})
+		testEnv.validatePodCache(t, "after-assume", testNode.Name, pod, expectedBindings, nil)
```

Comment:
This is because we begin with `nil` pod binding decision for every `FindPodVolumes` call now. Previously, `decision.provisionings` will be assigned an empty slice in `AssumePodVolumes` and not be cleared.